README.md

```diff
@@ -25,67 +27,83 @@ Upon deploying the Auction Contract you need to supply the `contract package has
 - `"token_contract_hash"`: Key-hash type of the contract package hash of the NFT contract
 - `"kyc_package_hash"`: Key-hash type of the contract package hash of the KYC contract
 - `"format"`: String/Text data to tell the contract to run an english or a dutch auction (only ENGLISH or DUTCH are valid data (capitalization is necessary))
-- `"starting_price"`: Option<U512> type. Dutch auction starting price. English auction doesn't use this so it requires this to be None or will fail to deploy. 
+- `"starting_price"`: Option<U512> type. Dutch auction starting price. English auction doesn't use this so it requires this to be None or will fail to deploy.
 - `"reserve_price"`: U512 type of reserve price aka smallest permitted selling price.
 - `"token_id"`: String/Text id of the NFT token put up for auction.
 - `"start_time"`: u64 UNIX timestamp of auctions starting time.
-- `"cancellation_time"`:  u64 UNIX timestamp of the latest time bids on the auction can be cancelled.
-- `"end_time"`:  u64 UNIX timestamp of the time the auction will end.
+- `"cancellation_time"`: u64 UNIX timestamp of the latest time bids on the auction can be cancelled.
+- `"end_time"`: u64 UNIX timestamp of the time the auction will end.
```

Comment:
I guess we forgot to update it as arguments were added, but the arguments should be listed here

Reply:
I added the missing arguments and changed versions. Let me know If I missed anything thanks.